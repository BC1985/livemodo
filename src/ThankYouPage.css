.push {
  height: 500px;
}

.hero {
  margin-bottom: -500px;
}
