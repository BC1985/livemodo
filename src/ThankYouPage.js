import React from "react";

export default function ThankYouPage() {
  return (
    <div className="hero">
      <h1>Thank You!</h1>
      <div className="redirected">
        <h3>You will be redirected in a few seconds...</h3>
      </div>
    </div>
  );
}
