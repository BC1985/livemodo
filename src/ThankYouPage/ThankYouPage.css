.push {
  height: -250px;
}

.redirected {
  margin-bottom: 250px;
}

h3 {
  text-align: center;
}
