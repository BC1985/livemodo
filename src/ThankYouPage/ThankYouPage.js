import React, { Component } from "react";
import { withRouter } from "react-router-dom";
import "./ThankYouPage.css";
class ThankYouPage extends Component {
  componentDidMount() {
    const redirect = () => {
      this.props.history.push("/");
    };
    setTimeout(redirect, 3000);
  }

  render() {
    return (
      <div className="thank-you-container">
        <div className="hero">
          <h1>Thank You!</h1>
          <div className="redirected">
            <h3>You will be redirected in a few seconds...</h3>
          </div>
        </div>
        <div className="thank-you-push" />
      </div>
    );
  }
}

export default withRouter(ThankYouPage);
