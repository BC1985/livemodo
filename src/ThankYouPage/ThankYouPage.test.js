import React from "react";
import ReactDOM from "react-dom";
import ThankYouPage from "../ThankYouPage/ThankYouPage";
import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <MemoryRouter>
      <ThankYouPage />
    </MemoryRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it("renders UI as expected", () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <ThankYouPage />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
