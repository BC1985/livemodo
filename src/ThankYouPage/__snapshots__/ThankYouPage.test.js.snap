// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI as expected 1`] = `
<div
  className="thank-you-container"
>
  <div
    className="hero"
  >
    <h1>
      Thank You!
    </h1>
    <div
      className="redirected"
    >
      <h3>
        You will be redirected in a few seconds...
      </h3>
    </div>
  </div>
  <div
    className="thank-you-push"
  />
</div>
`;
